packages/react-search-ui/src/containers/Result.js

```diff
@@ -76,14 +82,17 @@ export class ResultContainer extends Component {
     const { result, titleField, urlField, view } = this.props;
     const View = view || Result;
 
+    const safeResult = getSafeResult(result)
+
     return View({
```

Comment:
I feel like the API we are giving here is a little awkward.  I don't yet have a really strong opinion here, but I'll share my concerns.

1. fields and result serve basically the same purpose.  fields is arguably more friendly but I worry about the ambiguity of mixing raw/snippet/array/joined output and how we explain that to users.
2. safe continues to be a difficult term for me in this context — I'd probably prefer something like "escaped."  safe feels like we are extending our reach into risky areas by making assumptions about how someone is going to use the output.  We are probably right in most cases, though.

A good question to ask it — why do we need fields if we have result?